.github/workflows/build.yml

```
@@ -26,6 +26,9 @@ defaults:
 jobs:
 
   build:
+    concurrency:
+      group: build-${{ strategy.job-index }}-${{ github.event_name == 'pull_request' && github.ref || github.sha }}
```

Maybe better in case we cut-and-paste to other workflows?   Similar other 2 places.

```suggestion
      group: ${{ github.workflow }}-${{ strategy.job-index }}-${{ github.event_name == 'pull_request' && github.ref || github.sha }}
```